lib/src/modal_page/sliver_wolt_modal_sheet_page.dart

```diff
@@ -105,6 +105,9 @@ class SliverWoltModalSheetPage {
   /// the main content inside the [CustomScrollView] of the modal sheet.
   final List<Widget> mainContentSlivers;
 
+  /// Just like [mainContentSlivers] but exposes enclosing [BuildContext]
```

Comment:
Can you add more comment to `mainContentSlivers` to point out this detail and suggest using this? 

How about also adding `deprecated` tag to `mainContentSlivers` with a hint to use `mainContentSliversBuilder` instead?